src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -76,6 +76,21 @@ static inline unsigned uct_rc_verbs_iface_post_recv(uct_rc_verbs_iface_t *iface,
     return uct_rc_verbs_iface_post_recv_always(iface, count);
 }
 
+static void uct_rc_verbs_handle_failure(uct_rc_verbs_ep_t *ep,
+                                        uct_rc_verbs_iface_t *iface,
+                                        struct ibv_wc *wc)
+{
+    extern ucs_class_t UCS_CLASS_NAME(uct_rc_verbs_ep_t);
+    if (ep != NULL) {
```

Comment:
i guess we should call completion handlers with error status here?
Maybe instead of calling the original handlers (e.g get_bcopy) with status, we can just make few function pointer checks and call only mpool_put. and user completion if exists? this way we could avoid the need to add checks in the original handlers (e.g get_bcopy, atomics)